### DISCUSSION TITLE
Email Newsletter

### DISCUSSION THREAD

**Discussion by quicksnap** [supabase/supabase#1575] [Feature Requests]
Would love a Supabase newsletter so I can know when blog entries land :)

**Comment by inian** [comment]
Hey @quicksnap, here is the rss feed you can subscribe for blog posts - https://supabase.io/rss

**Reply by quicksnap** [reply]
woot! Now if only I used RSS... =P

**Comment by awalias** [comment]
hey @quicksnap we do send a monthly update email to all of our registered users, I can start adding a section for new blog posts. It looks a bit like this and usually goes out around the 3rd of each month

![image](https://user-images.githubusercontent.com/458736/118577564-3886ea80-b7bd-11eb-89a6-599c26d5e133.png)

**Reply by quicksnap** [reply]
Oh, nice! Thanks!
